api/v1/mapping/src/commonMain/kotlin/Mappings.kt

```diff
@@ -58,6 +58,7 @@ import org.eclipse.apoapsis.ortserver.api.v1.model.OrtRunFilters as ApiOrtRunFil
 import org.eclipse.apoapsis.ortserver.api.v1.model.OrtRunStatus as ApiOrtRunStatus
 import org.eclipse.apoapsis.ortserver.api.v1.model.OrtRunSummary as ApiOrtRunSummary
 import org.eclipse.apoapsis.ortserver.api.v1.model.Package as ApiPackage
+import org.eclipse.apoapsis.ortserver.api.v1.model.PackageCurationData as ApiPackageCurationData
```

Comment:
Commit message:

> include the applied packages in the result

Should this say "original packages"? Or "applied curations"?

BTW, in case we'd want to show original / uncurated data in the UI, we should be able to get it. I.e. our previous approach in ORT to "unapply" curations did not work, so also here we should probably get the original package plus any curations, and maybe for convenience also the curated package.

Reply:
Fixed the commit message. The package is already curated, if you think we want to show the uncurated package in the UI I can make another PR to add the uncurated package.

Reply:
> if you think we want to show the uncurated package in the UI I can make another PR to add the uncurated package.

I don't have an immediate need for that *right now*, but I remember you saying you'd want a similar step-by-step view of applied curations as in the workbench. And I still like that idea.

Reply:
I think the step-by-step view would require re-implementing ORT's logic to apply curations in the UI, but I will add the uncurated package, then the UI could at least show a diff between curated and uncurated package.